test/request.js

```diff
@@ -2294,26 +2337,17 @@ describe('Request', () => {
                 method: 'POST'
             };
 
-            await new Promise(async (resolve) => {
-
-                const req = Http.request(options, (res) => {
-
-                    expect([503, 408]).to.contain(res.statusCode);
-                    expect(timer.elapsed()).to.be.at.least(80);
-                    resolve();
-                });
+            const res = await new Promise((resolve, reject) => {
 
-                req.on('error', (err) => {
-
-                    expect(err).to.not.exist();
```

Comment:
In some (flaky) cases this error was being triggered late and causing the next test to fail.